<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_settings</class>
 <widget class="QDialog" name="Dialog_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_Settings">
     <property name="locale">
      <locale language="Walloon" country="Belgium"/>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Restore_Default">
     <property name="text">
      <string>Restore Default</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget_Settings">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="General_Settings">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Gen_1">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Gen_pho">
           <property name="text">
            <string>ρₙ(λ)</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QLineEdit" name="lineEdit_Gen_pho"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Gen_2">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Gen_Gamut">
           <property name="text">
            <string>Standard Gamut</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Gen_Gamut">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sRGB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Adobe RGB</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HTC Vive Pro Eye</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Oculus Rift</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Oculus Quest</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Oculus Quest 2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Gen_3">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Gen_illuminant">
           <property name="text">
            <string>Standard Illuminant</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Gen_illuminant">
           <item>
            <property name="text">
             <string>D65</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D50</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>E</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_Gen_4">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Gen_RGB">
           <property name="text">
            <string>RGB Values</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Gen_RGB">
           <item>
            <property name="text">
             <string>0 ... 255</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0 ... 1</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Plot_Settings">
      <attribute name="title">
       <string>Plot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_1">
         <item>
          <widget class="QLabel" name="label_Plot_Resolution">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Plot_Resolution"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_illuminant">
           <property name="text">
            <string> Illuminant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Plot_illuminant"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_Gamut">
           <property name="text">
            <string>Gamut</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Plot_Gamut"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_2">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance">
           <property name="text">
            <string>Reflectance Plot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy">
           <property name="text">
            <string>CIExy Plot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_3">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Width">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_Reflectance_Width"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Width_pixels">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Width">
           <property name="text">
            <string>Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_CIExy_Width"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Width_pixels">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_4">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Height">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_Reflectance_Height"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Reflectance_pixels">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Height">
           <property name="text">
            <string>Height</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_CIExy_Height"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Height_pixels">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_5">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Insert_Title">
           <property name="text">
            <string>Insert Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Plot_Reflectance_Insert_Title"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Insert_Title">
           <property name="text">
            <string>Insert Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Plot_CIExy_Insert_Title"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_6">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Title">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_Reflectance_Title"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Title">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Plot_CIExy_Title"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Plot_7">
         <item>
          <widget class="QLabel" name="label_Plot_Reflectance_Insert_Legend">
           <property name="text">
            <string>Insert Legend</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Plot_Reflectance_Insert_Legend"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Plot_CIExy_Insert_Legend">
           <property name="text">
            <string>Insert Legend</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Plot_CIExy_Insert_Legend"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Export_Settings">
      <attribute name="title">
       <string>Export</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Exp_1">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Export_Separator">
           <property name="text">
            <string>Export Number Separator</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Export_Separator">
           <item>
            <property name="text">
             <string>Comma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Point</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Exp_2">
         <item alignment="Qt::AlignmentFlag::AlignRight">
          <widget class="QLabel" name="label_Copy_Header">
           <property name="text">
            <string>Copy Header to Clipboard</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignLeft">
          <widget class="QComboBox" name="comboBox_Copy_Header">
           <item>
            <property name="text">
             <string>Yes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>No</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_Settings</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_Settings</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
